functional-tests/src/test/groovy/FrontendCore/spec/Contacts.groovy

```diff
@@ -62,13 +62,15 @@ class  Contacts extends GebReportingSpec {
         and: "I click on the contact page link."
         waitFor() { createContactButton.click() }
 
-        and: "I fill out the first form."
+        and: "I fill out the create contacts forms."
         createContactForm.createContactFormOne(input1)
         createContactForm.createContactFormTwo(input2)
-        waitFor {toastMessage != null}
+        waitFor {toastMessage != null}       
```

Comment:
It's a good practice to auto-trim trailing whitespace. This can usually be done on-save by your editor. Maybe check your settings and see if you can toggle that option on.

Reply:
weird the setting is turned on.

Reply:
fixed